packages/networks/src/networks.ts

```diff
@@ -63,6 +63,63 @@ export const networks = {
     protocolMultisig: undefined,
   },
   // TEST NETWORKS
+  mantleTestnetSepolia: {
+    name: 'Mantle Testnet Sepolia',
+    type: NetworkType.testnet,
+    key: 'mantleTestnetSepolia',
+    shortKey: 'mantleTestnet',
```

Comment:
This should probably be `mantleSepolia`. I noticed that we were not too consistent with the short name networks. I think the short name should have the form: `${network}${chainName}`

1. mantleGoerli
2. mantleSepolia
3. zoraGoerli
4. zoraSepolia
5. etc...

Open to discussion. If you agree, only change the name for the new networks. Changing older ones we will have to plan out more.

Reply:
hmm yeah i thought it made sense to keep the short-name consistent with the old testnet versions. all the other testnet shortkeys are just `<networkName>Testnet`. it's not too big of a deal for me because the shortkey doesn't really have a well defined usecase, but idk. i can update it, but i think it might make sense to do that if we update all of them at once including in the other repos that might be using shortKey for the older testnets